packages/experimental/Button/src/Button.styling.ts

```diff
@@ -32,18 +46,34 @@ export const stylingSettings: UseStylingOptions<ButtonProps, ButtonSlotProps, Bu
       (tokens: ButtonTokens, theme: Theme) => ({
         style: {
           color: tokens.color,
+          marginLeft: tokens.spacingIconContent ?? 1,
+          marginRight: -1,
```

Comment:
The negative margin here is to force the text to be laid out in the center of the button. There's an issue with the Segoe UI font such that the space around the text is asymmetric, so that the text is laid out off center in the text component:
![image](https://user-images.githubusercontent.com/4602628/134752698-9384e5a7-08f8-4462-b64a-b82061a9c999.png)

There's some workarounds that web has used that I will look into, but this may be what we have to do in RN to make the font render well.

Reply:
I extracted the constant into a function and left a comment on the function.